client/driver/java.go

```diff
@@ -133,8 +133,17 @@ func (d *JavaDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle,
 	// Get the environment variables.
 	envVars := TaskEnvironmentVariables(ctx, task)
 
+	args := []string{}
+	// Look for jvm options
+	jvm_options, ok := task.Config["jvm_options"]
+	if ok && jvm_options != "" {
+		d.logger.Printf("[DEBUG] driver.java: found JVM options: %s", jvm_options)
+	}
+
+	// Build the argument list
+	args = append(args, "-jar", filepath.Join(allocdir.TaskLocal, fName))
```

Comment:
@catsby that's my bad. Between the rebases and merge conflicts, I deleted it by mistake. Adding back.